### ISSUE TITLE
Droidchum on a Samsung J3

### ISSUE THREAD

**Issue by YungReymond** [illuminatedwax/pesterchum#241]
So i got this issue with my new phone, i cannot access the menu (opened windows, toggle quirks, that stuff), is it possible to have a way to use it again? It's really annoying...

**Comment by arcaneAgilmente** [comment]
No, no-one here made it and its old as hell. Use a regular irc client and
connect to the server that way.
